### PR TITLE
Starlight

### PR DESCRIPTION
**Pull request by JustAdler**
## Short description

Brigmedic glasses for DT because they'd get flashed with their sechud

## Checks

- [ ] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Aedler
- tweak: Changed deathsquad sechud to brigmedic glasses.

### CODE REVIEW

**Review by JustAdler — COMMENTED** [review]
I just realized now that deathsquad mask protects against flashes... maybe just a good upgrade from a sechud?

File: Resources/Prototypes/Roles/Jobs/CentComm/deathsquad.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     jumpsuit: ClothingUniformJumpsuitDeathSquad
     back: ClothingBackpackDeathSquad
     mask: ClothingMaskGasDeathSquad
-    eyes: ClothingEyesHudSecurity
+    eyes: ClothingEyesGlassesBrigmedic
```
Comment (Rinary1) [inline Resources/Prototypes/Roles/Jobs/CentComm/deathsquad.yml]: am? Huh, better to add death squad a omni eye implant or something else

**Review by Rinary1 — CHANGES_REQUESTED** [review]

**Comment by JustAdler** [comment]
Added an implant, but i don't know what to do with it. Its my first PR

**Comment by Rinary1** [comment]
> Added an implant, but i don't know what to do with it. Its my first PR

unluck, but you need to use C# for adding Implants addition in start gear, maybe i write this thing in feuture.

**Pull request closed without merging**